src/Fusion.Summary.Api/Controllers/DepartmentsController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Fusion.Summary.Api.Database.Entities;
+using Fusion.Summary.Api.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Fusion.Summary.Api.Controllers;
+
+public record PutDepartmentRquest(string DepartmentSapId, string FullDepartmentName, Guid ResourceOwnerAzureUniqueId);
+
+public record GetDepartmentResponse(string departmentSapId, Guid resourceOwnerAzureUniqueId, string fullDepartmentName);
+
+[Authorize]
+[Route("api/[controller]")]
```

Comment:
Should align with route patterns established in existing apis and throughout the core apis. 

`[HttpGet("/departments")]` e.g. 

The convention based pattern is too volitile to internal refactoring. When we expose an endpoint we are responsible for maintaining it properly, like versioning and deprecation.